cloud: add windows ca path (#11563)
pingcap/docs: tidb-cloud/secure-connections-to-serverless-tier-clusters.md

> However, TiDB Serverless Tier ensures always using a CA root certificate that is commonly available, which is provided in all common systems.
>
> If you really need the CA certificate of a TiDB Serverless Tier cluster, it is recommended that you download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) instead of the single CA certificate in case we change the CA in the future.
>

> > If you are using some GUI clients, like DBeaver, which does not accept a cert file with multiple certs inside. In such a case, you have to download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate.

Reviewer: ```suggestion
> If you are using a GUI client, such as DBeaver, which does not accept a certificate file with multiple certificates inside, you must download the [ISRG Root X1](https://letsencrypt.org/certs/isrgrootx1.pem.txt) certificate.
```

/etc/ssl/ca-bundle.pem
```

**Windows**

> Windows does not offer a specific path to CA root, instead, it uses [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.

Reviewer: ```suggestion
Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.
```

> 

Reviewer: Can we provide more detailed steps? For example, how can I point at this file for the drivers?

Reviewer 2: It's the same as other OS. How to point it is drivers' work and we have already described it in previous section.

Reviewer: Got it. Can you please link to that section? You know, people don't always read docs linearly and might skip some sections and thus get frustrated.

> Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, we recommend downloading the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) as the CA certificates and pointing at this file for the drivers.

Reviewer: How about this? It's easier to follow instructions. @zhangyangyu 

```suggestion
Windows does not offer a specific path to the CA root. Instead, it uses the [registry](https://learn.microsoft.com/en-us/windows-hardware/drivers/install/local-machine-and-current-user-certificate-stores) to store certificates. For this reason, to specify the CA root path on Windows, take the following steps:

1. Download the [Mozilla CA Certificate bundle](https://curl.se/docs/caextract.html) and save it in a path you prefer, such as `<path_to_mozilla_ca_cert_bundle>`.
2. Use the path (`<path_to_mozilla_ca_cert_bundle>`) as your CA root path when you connect to a Serverless Tier cluster.
```

Reviewer 2: LGTM
